tools/test/memap/memap_test.py

```diff
@@ -191,8 +201,10 @@ def test_generate_output_csv_ci(memap_parser, tmpdir, depth):
     """
     Test ensures that an output of type "csv-ci" can be generated correctly
 
-    :return:
+    :param memap_parser: Mocked parser
+    :param tmpdir: a temporary directory
```

Comment:
Might be better to say why there is even a temporary directory? Otherwise not that helpful a parameter description.

Reply:
Yes of course.